src/Illuminate/Database/Console/Factories/stubs/factory.stub

```diff
@@ -2,6 +2,8 @@
 
 use Faker\Generator as Faker;
 
+/* @var Illuminate\Database\Eloquent\Factory $factory */
```

Comment:
This is not actually phpdoc. There needs to be two stars at the start. This is just a comment, and it's actually a bug if any IDE processes it as PHPDoc.

Reply:
Using a single asterisk and putting the variable name before the type. From \Drupal\node\Tests\NodeTokenReplaceTest::testNodeTokenReplacement:
 /* @var $node \Drupal\node\NodeInterface */
This format is supported by Eclipse, Netbeans, PHPStorm and Zend Studio. If /** is substituted, orif the order is reversed it does not work in Netbeans (not sure about the other IDEs) as of this time (July 2014). https://www.drupal.org/node/2305593

Reply:
I'm not saying that I am right with the choice to use a single asterisk, I'm just saying that it was intentional and this is the reason for it.

Reply:
Drupal is just plain wrong here. The single star is T_COMMENT and double is T_DOC_COMMENT.